### PR TITLE
Perform module PR additions directly to installed modules location

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: lib/katello_deploy/module_pull_request.rb [hunk 1]
```diff
@@ -9,8 +9,7 @@ def initialize(args)
 
     def prepare
       install_git
-      bundle_install
```
Comment (stbenjam) [inline lib/katello_deploy/module_pull_request.rb]: Why'd this get removed?

Comment (stbenjam) [inline lib/katello_deploy/module_pull_request.rb]: Oh I see, you git checkout into the katello-installer-base directory's modules.

**Comment by stbenjam** [comment]
What about if the pipeline has an oil spill and we're out of commission for a while, the installer package could be quite out of date with the git repos.  Do you think it matters if we have the latest and greatest checkouts + the PR's?

**Comment by ehelms** [comment]
There is sort of two things working in conjunction here. If there is a local installer, the symlinking occurs (https://github.com/Katello/katello-deploy/blob/master/lib/katello_deploy/installer.rb#L42) which means whatever modules you had in kdeploy/katello-installer/modules are your base set of modules that the PRs then get applied in conjunction with. If you do not have the local installer, then it will apply them to the modules installed via the RPM.

**Comment by stbenjam** [comment]
Ah I see, I've tried both ways now and works great.  Happy with this once tests pass.

**Comment by ehelms** [comment]
@stbenjam tests are now passing

**Comment by stbenjam** [comment]
ACK

**Pull request merged**